src/bicep/modules/policyAssignment.bicep

```diff
@@ -0,0 +1,42 @@
+targetScope = 'resourceGroup'
```

Comment:
can remove, the default scope is `resourceGroup`

Reply:
Should this be removed is clients deploy separately via pointing to this file and using create sub deployment? Seems like each module should scope correctly no matter the calling bicep scope?

Reply:
Ah, is that the intent? I think it has to be one or the other. An `az deployment sub create` would not start unless the targetScope is `subscription` and you'd get an error similar to `The target scope "resourceGroup" does not match the deployment scope "subscription".`

Reply:
I've tested running policyAssignment.bicep it and it works, the caveat is run `as group create` and include operations resource group name to scope the LA lookup + the resource group name you want to deploy the policy to. Maybe its easier to just use the LA WS ID but this way I can look up details since each policy initiative can base LA on name, ID or WS ID separately I can get the correct values as well as get primary get when needed.